xwiki-platform-core/xwiki-platform-test/xwiki-platform-test-ui/src/main/java/org/xwiki/test/ui/XWikiWebDriver.java

```diff
@@ -649,6 +649,24 @@ public WebElement scrollTo(WebElement element)
         return element;
     }
 
+    /**
+     * Scrolls to the given coordinates inside the web page. If you want to scroll to a specific {@link WebElement}, see
+     * {@link #scrollTo(WebElement)}.
+     *
+     * @param xCoord is the pixel along the horizontal axis of the web page that you want displayed in the upper
+     *     left
+     * @param yCoord is the pixel along the vertical axis of the web page that you want displayed in the upper left
+     * @see <a href="https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollTo">MDN Web Docs -
+     *     Element.scrollTo()</a>
+     * @see #scrollTo(WebElement)
+     * @since 13.3RC1
+     * @since 12.10.7
+     */
+    public void scrollTo(int xCoord, int yCoord)
```

Comment:
Hmm my last comment was proposing to add a `scrollToTop` but if you think this one add more value I'm ok to add it.

Reply:
No no, I agree with you actually, just a misunderstanding.
I'll add it too but that does not prevent to keep this one too.

Reply:
`scrollToTop` added https://github.com/xwiki/xwiki-platform/pull/1598/files#diff-3f695424dbbf520ad4644d8d288aafd9ee5d18652641363395e48112b3b1e55eR258